package.json

```diff
@@ -30,11 +30,10 @@
     "@envelop/graphql-jit": "^1.1.1",
     "@envelop/parser-cache": "^2.2.0",
     "@envelop/validation-cache": "^2.2.0",
-    "@faststore/api": "^1.8.4",
-    "@faststore/sdk": "^1.8.4",
-    "@faststore/ui": "^1.8.4",
-    "@reach/router": "^1.3.4",
```

Comment:
Why are we removing it? 👀  It seems like we are still using it on [useProductLink.ts](https://github.com/vtex-sites/gatsby.store/blob/b0b2f8464e0177eef8dfe39faf6049901e299709/src/sdk/product/useProductLink.ts#L3)

Reply:
follow-up: https://github.com/vtex-sites/gatsby.store/pull/48